service/src/test/resources/ehr/mapper/diagnosticreport/observation/observation_compound_statement_with_multiple_interpretations.xml

```diff
@@ -0,0 +1,51 @@
+<component typeCode="COMP" contextConductionInd="true">
+    <CompoundStatement classCode="CLUSTER" moodCode="EVN">
+        <id root="some-id"/>
+        <code nullFlavor="UNK">
+    <originalText>test</originalText>
+</code>
+        <statusCode code="COMPLETE"/>
+        <effectiveTime>
+            <center value="20190601113000"/>
+        </effectiveTime>
+        <availabilityTime value="20190601113000"/>
+        <component typeCode="COMP" contextConductionInd="true">
+    <ObservationStatement classCode="OBS" moodCode="EVN">
+        <id root="some-id"/>
+        <code nullFlavor="UNK">
+    <originalText>test</originalText>
+</code>
+        <statusCode code="COMPLETE"/>
+        <effectiveTime>
+            <center value="20190601113000"/>
+        </effectiveTime>
+        <availabilityTime value="20190601113000"/>
+            <interpretationCode code="HI" codeSystem="2.16.840.1.113883.2.1.6.5" displayName="Above high reference limit">
+    <originalText>Very High Value</originalText>
+</interpretationCode>
+
+    </ObservationStatement>
+</component>
+            <component typeCode="COMP" contextConductionInd="true">
+    <NarrativeStatement classCode="OBS" moodCode="EVN">
+        <id root="5E496953-065B-41F2-9577-BE8F2FBD0757"/>
+        <text mediaType="text/x-h7uk-pmip">CommentType:USER COMMENT
+            CommentDate:20190601113000
+
+            Test comment</text>
+        <statusCode code="COMPLETE"/>
+        <availabilityTime value="20190601113000"/>
+    </NarrativeStatement>
+</component><component typeCode="COMP" contextConductionInd="true">
+    <NarrativeStatement classCode="OBS" moodCode="EVN">
+        <id root="5E496953-065B-41F2-9577-BE8F2FBD0757"/>
+        <text mediaType="text/x-h7uk-pmip">CommentType:LABORATORY RESULT DETAIL(E136)
+            CommentDate:20190601113000
+
+            Example without userSelected</text>
```

Comment:
Shouldn't here be **Interpretation: Example without userSelected** ? isn't it from interpretation.coding[0].display ?

Reply:
We do prepend "Interpretation: " for diagnosticReport.conclusion, but I don't think we should do it for observation.interpretation (from looking at the mapping spreadsheet).

See row 156 in DiagnosticReport
```
interpretation |
As per Observation (Uncategorised) but with the distinction that any interpretation that would previously have generated a value string for inclusion in pertinentInformation text should generate a NarrativeStatement containing the value text of comment type  LABORATORY RESULT DETAIL(E136) and date the same as the generated ObservationStatement/effectiveTime to be included within the CompoundStatement generated from the Observation
```

Doesn't say anything about prepending "Interpretation: ", unless you can see it somewhere else?

Reply:
We do prepend "Interpretation: " for diagnosticReport.conclusion, but didn't realise we should do it for observation.interpretation as well. Can make that change now